driver-core/src/main/java/com/datastax/driver/core/Metrics.java

```diff
@@ -273,6 +273,7 @@ void shutdown() {
         private final Counter writeTimeouts = registry.counter("write-timeouts");
```

Comment:
Weird//slightly offtopic topic, but what is the policy on adding methods to classes that aren't meant to be extended/used otherwise?  Can we add methods to those (i.e. Metrics) without breaking API, i.e. will clirr care (guessing it doesn't because we can build)?  Should we make Metrics final in 3.0 just to avoid this possibility (can't remotely imagine this ever being a problem anyways).